base/applications/network/ftp/CMakeLists.txt

```diff
@@ -12,12 +10,14 @@ list(APPEND SOURCE
     precomp.h)
 
 add_executable(ftp ${SOURCE} ftp.rc)
+target_compile_definitions(ftp PRIVATE lint)
+
 set_module_type(ftp win32cui)
 add_importlibs(ftp ws2_32 iphlpapi msvcrt kernel32)
+target_link_libraries(ftp oldnames)
```

Comment:
I think as a rule, we want `target_link_libraries` before `add_importlibs` (since libraries will often depend on things from dlls)

Reply:
Yes, but stuff from msvcrtex depends on it :-(